database/migrations/create_moonguard_tables.php.stub

```diff
@@ -22,6 +22,10 @@ class CreateMoonGuardTables extends Migration {
             $table->unsignedInteger('max_request_duration_ms')->default(1000);
             $table->timestamp('down_for_maintenance_at')->nullable();
             $table->string('api_token', 60)->nullable();
+            $table->integer('cpu_limit')->nullable();
+            $table->integer('ram_limit')->nullable();
+            $table->integer('disk_limit')->nullable();
+            $table->boolean('monitoring_notification_enabled')->default(false);
```

Comment:
Maybe this column should be renamed into `hardware_monitoring_notification_enabled`. The concept of Monitoring is too big and ambiguos. It should be clear for users and devs to understand what is this column for.